tmpl/funcmaps.go

```diff
@@ -544,3 +547,24 @@ func convertAmountInCentsToPercent(totalAmount, amount uint64) float64 {
 func convertAmountInCentsToPercentPerDay(totalAmount, amount uint64) float64 {
 	return float64(amount) * 30 * 100 / float64(totalAmount)
 }
+
+func getCustomId(serviceUser string) string {
+	switch {
+	case serviceUser == "onev2":
+		return "GATEWAY"
+	case serviceUser == "pagarme":
+		return "PSP"
+	default:
+		return ""
+	}
+}
+
+func getWebhookUrl() string {
+	return config.Get().BaasWebhookUrl
+}
+
+func getValue(percentageStr string) int64 {
+	numericalPart := strings.Replace(percentageStr, ".", "", -1)
```

Comment:
Opa, aqui podemos usar o **ReplaceAll**, não ? 😀 
```suggestion
	numericalPart := strings.ReplaceAll(percentageStr, ".", "", -1)
```

Reply:
Pode sim! Vou alterar aqui!